NTUEELightDance/LightDance-Editor
# EDITOR-#356. Add new commands in utils, fix minor issues

### What is this PR for?
* Add new commands in `utils/package.json` (under utils directory)
  * init backend DB: `yarn initDB` -> will execute `node initDB.js ./out/exportData.json`
  * saveData from backend DB: `yarn save` -> will execute `node saveData.js ./out/exportData.json`
  * code style fix: `yarn lint`
* Delete all files in relation to LED (while saved a backup file of older version)

### What type of PR is it?

[Bug Fix | Improvement | Feature | Documentation | Hot Fix | Refactoring | Chore]
Feature

### Todos

* [x] Add initDB shortcut command
* [x] Add saveData shortcut command
* [x] Add eslint rule (same as in editor-server)

### What is the Github issue?
#356 

### How should this be tested?
1. Execute `yarn install:all` (added eslint in utils)
2. Execute `yarn start` in `editor-server/`, make sure the backend is running
3. Test initDB: execute `yarn initDB` 
4. Test saveData: execute `yarn save` -> if success, the newest data will be stored in `./out/exportData.json`

### Screenshots (if appropriate)

### Questions

* Do the license files need updating? No
* Are there breaking changes for older versions? No
* Does this need new documentation? No
